app/components/ui/Dropdown.tsx

```diff
@@ -0,0 +1,48 @@
+import { type ReactNode } from 'react';
+import { Transition } from '@headlessui/react';
+
```

Comment:
since all the other components are using radix ui, can you adapt the same. otherwise it will soon bloat with multiple ui frameworks

Reply:
and is its only for animation transition then try framer motion